docker/intel-dgpu-driver.Dockerfile

```diff
@@ -37,10 +38,17 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/drm .config \
     && make olddefconfig && make -j $(nproc) && make modules_install
 
+# Create /build/modules directory with all *.ko and *.ko.xz files
+RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -type f | xargs -I {} cp {} /build/modules/ \
```

Comment:
Mabe below command is more concise, and can do the same job
RUN mkdir -p /build/modules && \
find /lib/modules/${KERNEL_FULL_VERSION}/ \
-name "*.ko" -o -name "modules.order" -o "modules.builtin" -type f \
| xargs -I {} install {} /build/modules/

Reply:
Please be noted the continuous character  is removed by github. :-)

Reply:
> Mabe below command is more concise, and can do the same job RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -o -name "modules.order" -o "modules.builtin" -type f | xargs -I {} install {} /build/modules/

Interesting, I can try this out :)

Reply:
`RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -o -name "*.ko.xz" -o -name "modules.order" -o -name "modules.builtin" -type f | xargs -I {} install -D {} /build/modules/` worked. PR updated.